contrib/go-redis/redis/redis.go

```diff
@@ -133,6 +148,16 @@ func commandsToString(cmds []redis.Cmder) string {
 	return b.String()
 }
 
+// Pipelined executes a function parameter to build a Pipeline and then immediately executes the built pipeline.
+func (c *Pipeliner) Pipelined(fn func(redis.Pipeliner) error) ([]redis.Cmder, error) {
+	if err := fn(c); err != nil {
+		return nil, err
+	}
+	cmds, err := c.Exec()
+	_ = c.Close()
+	return cmds, err
```

Comment:
I know this was copied directly from `go-redis`, but I think we can clean it up a little. 
```suggestion
	defer c.Close()
	return c.Exec()
```

Reply:
Sounds great, thanks for the suggestion :)